ios/RNSScreenStack.mm

```diff
@@ -1117,9 +1117,8 @@ - (void)mountChildComponentView:(UIView<RCTComponentViewProtocol> *)childCompone
 
   [_reactSubviews insertObject:(RNSScreenView *)childComponentView atIndex:index];
   ((RNSScreenView *)childComponentView).reactSuperview = self;
-  dispatch_async(dispatch_get_main_queue(), ^{
-    [self maybeAddToParentAndUpdateContainer];
-  });
+  // Child update operation is performed in the mountingTransactionDidMount method to prevent error, while
+  // navigating to `n` different screens at the same time.
```

Comment:
I would rather have this comment removed or limited to following

```suggestion
  // Container update is done after all mount operations are executed in `- [RNSScreenStackView mountingTransactionDidMount: withSurfaceTelemetry:]`
```

Reply:
As when reading the code it is not really helpful and rather confusing.

Reply:
Fixed in https://github.com/software-mansion/react-native-screens/pull/2249/commits/3d89cfd66f04044c2fb3062765f7becd6dbe3f62.